### DISCUSSION TITLE
Unable to connect email account after brand new install of Elementary OS 7

### DISCUSSION THREAD

**Discussion by BruceH2020** [elementary/mail#947] [Q&A] [answered]
I've just installed a brand new Elementary OS 7 from scratch and now I'm having a problem setting-up email.

I started the email app and it pops up a dialog saying "Mail uses email accounts configured in system settings". That's fine. 

So I go to system settings, add an account, provide the necessary imap and smtp settings, the connection test succeeds and the account shows in "Online Accounts".

I launch Mail again, and it displays the same dialog. I click on the "Online Accounts..." button and it opens  the Online Accounts page of System Settings. And there is no visible way of selecting the mail account. Clicking or double-clicking has no effect.

Now what do I do?

I don't want to sound ungrateful but shouldn't the instructions be more obvious?

**Comment by leolost2605** [comment, marked as answer]
Hey @BruceH2020,
that certainly isn't how Mail should behave. Usually once at least one mail account got configured this dialog is supposed to go away, so there is definitely something wrong here. 
If possible please run Mail from terminal by first executing `killall io.elementary.mail`. Then run `io.elementary.mail` and post the output you get in the terminal here.
If you get no output from Mail it would be helpful if you could do the same with the System Settings app by replacing `io.elementary.mail` with `io.elementary.switchboard` but then also going through the process of adding your mail account again (if the output includes your Mail address you might want to redact it before posting it here, only your mail provider could be useful to know).

**Reply by BruceH2020** [reply]
Running io.elementary.mail from terminal started the app and it appeared to pickup the email account. However this one error message appeared about 5 times:

(io.elementary.mail:2790): Gtk-CRITICAL **: 08:32:47.991: _gtk_widget_get_preferred_size_for_size: assertion 'size >= 1' failed

However, I say 'appeared' because although it has picked up the mailboxes and has the right message counts, clicking on inbox does nothing and there is no list of messages. I've uploaded a screenshot to Flickr. [https://flic.kr/p/2oUdHH7](url)

Note:
- This is a normal user account, not administrator.
- I haven't tried re-adding the account using .switchboard yet. (Let me know if you still want me to do that.)
- The error message appears a few more times in the screen shot because I went into the control panel and made the font size bigger. (old eyes!)

**Reply by leolost2605** [reply]
When adding a new mail account unfortunately you currently have to refresh every folder once manually before messages in it start to appear automatically. To do this open mail, select the folder you want (try starting with the inbox) and click the refresh button on the bottom left corner (or hit F12). The button should start spinning now and after a while your messages will appear. From now on old and new messages appear automatically though.
I hope this fixes your issue, if not please reach out again :)

**Reply by BruceH2020** [reply]
Thanks leolost - I've marked your first answer as the one that fixed it.
